libs/product/src/drivers/magento/queries/fragments/configurable-product.ts

```diff
@@ -0,0 +1,45 @@
+import gql from 'graphql-tag';
+
+export const magentoConfigurableProductFragment = gql`
+  fragment magentoConfigurableProduct on ConfigurableProduct {
+		items {
+			configurable_options {
+				attribute_code
+				attribute_id
+				id
+				label
+				position
+				product_id
+				values {
+					label
+					value_index
+					swatch_data {
+						value
+						thumbnail
+					}
+				}
+			}
+			variants {
+				attributes {
+					code
+					label
+					value_index
+				}
+				product {
```

Comment:
I made product only retrieve price and an image, because I think that's all that will be needed and I'm trying to cut down on the query size

Reply:
I think you need the `sku` for the subsequent add to cart call.

I also think you won't have enough info to meet the `DaffProduct` type guarantee. Maybe consider shifting the type on the DaffConfigurableProduct down to a Partial<DaffProduct> or a special type with explicit type guarantees.

Reply:
Oh, I didn't know the sku worked that way. 

And yea, I realized this when I was working on the next PR. I made that change there, but I'll make it here since you bring it up